lib/data-helper.js

```diff
@@ -253,3 +253,29 @@ export function getCalypsoLiveQuery() {
 	}
 	return 'branch=' + config.get( 'branchName' );
 }
+
+export function getCalypsoURL( route = '', queryStrings = [] ) {
+	let queryString = '';
```

Comment:
is it obvious how `queryString` is different from `queryStrings`?

Reply:
Yes, I think it is clear enough